lib/queue_classic.rb

```diff
@@ -31,6 +31,10 @@ module QC
   # for more details.
   FORK_WORKER = !ENV["QC_FORK_WORKER"].nil?
 
+
+  #Delete the queue from the table after completion
+  DELETE_QUEUE = ENV["DELETE_QUEUE"] || true
+
```

Comment:
Doing so will affect all queues within the same process, better to do this on the initialize()